src/reporter/tests/run_tests.timescale.sh

```diff
@@ -5,12 +5,9 @@ docker build -t smartsdk/quantumleap ../../../
 docker-compose -f docker-compose.timescale.yml up -d
 sleep 10
 
-# Set Postgres port to same value as in docker-compose.timescale.yml
-export POSTGRES_PORT='54320'
```

Comment:
we'll have to put this line back in for now w/ a port value of `5432` otherwise pytest will skip some Timescale tests, see `test_timescale_types`. also, some of the skipped tests will fail b/c of the missing CRS in the returned GeoJSON---well, at least that;s what I think it should happen. So those tests will need some massaging...